modules/core/include/opencv2/core/cvdef.h

```diff
@@ -269,6 +269,8 @@ namespace cv {
 #define CV_CPU_AVX512_CLX       261
 #define CV_CPU_AVX512_ICL       262
 
+#define CV_CPU_RVV              300
```

Comment:
Please use 210 here and below.

256+ is used for "groups".

Reply:
Fixed. Thanks a lot for your review!